tests/pck/cora/144.0002.no_dcodes.json

```diff
@@ -9,6 +9,7 @@
     "0220": "UK National",
     "0230": "European countries",
     "0420": "Turnover increased by at least 10% due to merger with another business or part of it",
+    "0440": "No significant changes occurred",
```

Comment:
Remove either this or the 0420 above as it's not possible to get this state if it was from eq.
Also need to update the pck file to reflect this.